src/components/Nav/Desktop/index.tsx

```diff
@@ -20,20 +22,12 @@ const DesktopNavMenu = ({ toggleColorMode }: DesktopNavMenuProps) => {
   const { locale } = useRouter()
   const languagePickerRef = useRef<HTMLButtonElement>(null)
 
-  const ThemeIcon = useColorModeValue(<MdBrightness2 />, <MdWbSunny />)
+  const ThemeIcon = useColorModeValue(MdBrightness2, MdWbSunny)
```

Comment:
Not at all against this, but just noting an alternative way this can be done as we migrate to TW:

```tsx
<MdBrightness2 className="dark:hidden" />
<MdWbSunny className="not-[dark]:hidden" />
```

Can see where both approaches could come in handy, not sure I have a strong preference or desire to standardize it.

Reply:
Great idea! I haven't thought about that option, I like the no-js dependency. I'll test it out and let you know.